src/rust/engine/src/rule_graph.rs

```diff
@@ -668,51 +698,25 @@ impl<'t> GraphMaker<'t> {
     }) {
       vec![*param]
     } else {
-      // Group by the simplified version of each rule: if exactly one, we're finished. We prefer
-      // the non-ambiguous rule with the smallest set of Params, as that minimizes Node identities
-      // in the graph and biases toward receiving values from dependencies (which do not affect our
-      // identity) rather than dependents.
-      let mut rules_by_kind: HashMap<EntryWithDeps, (usize, &Entry)> = HashMap::new();
-      for satisfiable_entry in &satisfiable_entries {
+      // We prefer the non-ambiguous rule with the smallest set of Params, as that minimizes Node
+      // identities in the graph and biases toward receiving values from dependencies (which do not
```

Comment:
s/identifies/selections?

Reply:
This is "identities", which is accurate here... we refer to the `Params` that a `@rule` consumes as part of its "identity", because the same `@rule` can be used with multiple sets of `Params`, which (as seen in this ticket!) affects their behaviour.